run_xnat2bids.py

```diff
@@ -11,65 +13,115 @@
 
 
 logging.basicConfig(format='%(levelname)s: %(message)s', level=logging.NOTSET)
+logging.getLogger('asyncio').setLevel(logging.WARNING)
+
+# Define a coroutine function to run subprocess command
+async def run_subprocess(srun_cmd):
+    await asyncio.create_subprocess_exec(*srun_cmd)
```

Comment:
Do you need this? Can you gather the processes returned by `create_subprocess_exec`?